ntbs-service/Pages/Index.cshtml

```diff
@@ -14,6 +14,7 @@
     @if (Model.KpiFilter.Any())
     {
         <partial name="_HomepageKpiDetails" model="@Model"/>
+        <br/>
```

Comment:
Feels like we'd be better off with a css margin than a <br> here? Unless it's trying to achieve something else?